src/Data/HashMap/Linear.hs

```diff
@@ -57,25 +69,42 @@ import GHC.Stack
 -- * https://cs.uwaterloo.ca/research/tr/1986/CS-86-14.pdf
 --
 
+-- * Constants
+
```

Comment:
Not needed, but it would be nice if you could use the `-- # Constants` followed by some `-----`s just for consistency.

Reply:
Actually, it's probably needed. Haddock will probably freak out if it sees a `-- *` comment outside of an export list.